### DISCUSSION TITLE
APIのレートリミットはどのように返却されますか？

### DISCUSSION THREAD

**Discussion by myConsciousness** [misskey-dev/misskey#10023] [Q&A]
こんにちは。

Misskey APIについての質問なのですが、APIのレートリミットはレスポンスヘッダーに `x-rate-limit-remaining` といったフィールドに設定されて返却されるのでしょうか？ `/api/i` エンドポイントを試したところ、これらのレートリミットに関するヘッダー情報は確認できませんでした。

**Comment by syuilo** [comment]
現在レートリミットの詳細情報を返すようにはなっていません。
